docs/source/index.rst

```diff
@@ -148,3 +148,4 @@ For the entire Python API reference documentation, you can visit the docs here:
    tutorials
    troubleshooting
    cli_reference
+   manifest_submission
```

Comment:
nit: Shift this to be above troubleshooting

Reply:
makes sense. Thanks for pointing that out.